WalletLibrary/Submodules/VerifiableCredential-SDK-iOS/VCNetworking/VCNetworking/PresentationNetworkCalls.swift

```diff
@@ -3,15 +3,9 @@
 *  Licensed under the MIT License. See License.txt in the project root for license information.
 *--------------------------------------------------------------------------------------------*/
 
-import PromiseKit
-
-#if canImport(VCEntities)
-    import VCEntities
-#endif
-
 protocol PresentationNetworking {
-    func getRequest(withUrl url: String) -> Promise<PresentationRequestToken>
-    func sendResponse(usingUrl url: String, withBody body: PresentationResponse) -> Promise<String?>
+    func getRequest(withUrl url: String) async throws -> PresentationRequestToken
+    func sendResponse(usingUrl url: String, withBody body: PresentationResponse) async throws
```

Comment:
Same here, changed method signature?

Reply:
same as above. The String? returned is legacy code and nil was always returned anyway.